acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
Also, if you look at `beaker`'s 2.3.0 tag:
https://github.com/puppetlabs/beaker/blob/beaker2.3.0/lib/beaker/host/unix.rb#L61

This value is already available on the host defaults.